### DISCUSSION TITLE
Is Shared Files synchronization supported?

### DISCUSSION THREAD

**Discussion by rrodrigueznt** [abraunegg/onedrive#2118] [General]
While working with the current client release, if I set `sync_business_shared_folders = "true"` and no `business_shared_folders` file exists, I get a series of WARNING records in the log as these:
```
2022-Aug-30 15:51:23.7438046    WARNING: Not syncing this OneDrive Business Shared File: Anexo1_Indicadores_Cuantitativos.pdf
2022-Aug-30 15:51:23.7439561    WARNING: Not syncing this OneDrive Business Shared File: 15_Plan Viabilidade_IGFAE_draft.xlsx
2022-Aug-30 15:51:23.7441087    WARNING: Not syncing this OneDrive Business Shared File: AnnualReport2020_DraftV01.docx
2022-Aug-30 15:51:23.7442632    WARNING: Not syncing this OneDrive Business Shared File: ResearchProjects_05-27-2022.xlsx
2022-Aug-30 15:51:23.7444176    WARNING: Not syncing this OneDrive Business Shared File: 1695877_554229_Factura de orixe SEF.pdf
2022-Aug-30 15:51:23.744572     WARNING: Not syncing this OneDrive Business Shared File: Acuerdo IGFAE-IFIC.pdf
2022-Aug-30 15:51:23.7447255    WARNING: Not syncing this OneDrive Business Shared File: 5484_gl_anexo_plan.docx
```
The list includes files shared with me, but not all of them. Please, is this a side effect of not setting a `business_shared_folders` file? Could it be desirable to be able to synchronize shared files? At least it is for me! Thanks!

**Comment by rrodrigueznt** [comment]
I'm afraid I was wrong: the warning appears also when there is a `business_shared_folders` file on place. Sorry for the noise. It seems I arrived to a wrong conclusion.

The main question remains: is synchronization of OneDrive Business Shared Files supported? Thanks!

**Comment by abraunegg** [comment]
@rrodrigueznt 
> is synchronization of OneDrive Business Shared Files supported?

No it is not. Please refer to: https://github.com/abraunegg/onedrive/issues/1300

To support 'files':
* The existing feature needs to be 100% re-written

Why:
* Because the original feature request was for 'Shared Folders' .. not individual shared files ..
* Not enough user testing of the feature before it was implemented

When will it be 'fixed' ?
* The entire client needs a full re-write and will be considered as part of that
